jupyter_remote_desktop_proxy/setup_websockify.py

```diff
@@ -34,11 +34,11 @@ def setup_websockify():
         is_tigervnc = "tigervnc" in vncserver_file.read().casefold()
 
     if is_tigervnc:
+        websockify_args = ['--unix-target', sockets_path]
         vnc_args = [vncserver, '-rfbunixpath', sockets_path]
-        socket_args = ['--unix-target', sockets_path]
     else:
-        vnc_args = [vncserver, '-rfbport', '{port}']
-        socket_args = []
+        websockify_args = []
+        vnc_args = [vncserver, '-localhost', '-rfbport', '{port}']
```

Comment:
## About passing `-localhost` to turbovnc

This won't change anything, its just about making both VNC servers startup as localhost and not just TigerVNC for consistency.

>TigerVNC listens to localhost by default, and TurboVNC doesn't. Let's make them do the same.
>
> When websockify starts either TigerVNC or TurboVNC, it will listen only to localhost anyhow - so this doesn't change anything really but could reduce the complexity a bit when debugging these things, as I've done at length now.